### DISCUSSION TITLE
server certificate verification failed. CAfile: none CRLfile: none (128)

### DISCUSSION THREAD

**Discussion by hosnyben** [WeblateOrg/weblate#9469] [Q&A] [answered]
Hello community,
I have a self-hosted Weblate 4.18.1. I have an issue that we're not able to add repositories from my selfhosted gitlab. as it throws the ssl verification issue.

server certificate verification failed. CAfile: none CRLfile: none (128).

Is there a way I can bypass ssl verification of the git agent. Thank you.

**Comment by nijel** [comment, marked as answer]
You can modify Weblate git configuration in `DATA_DIR/home/.gitconfig`, see https://docs.weblate.org/en/latest/admin/config.html#data-dir.

**Reply by hosnyben** [reply]
It works ! Thank you !
